python/pyarrow/parquet/core.py

```diff
@@ -821,7 +823,10 @@ def _sanitize_table(table, new_schema, flavor):
     and should be combined with a compression codec.
 column_encoding : string or dict, default None
     Specify the encoding scheme on a per column basis.
-    Currently supported values: {'PLAIN', 'BYTE_STREAM_SPLIT'}.
+    Only if "use_dictionary" and "use_byte_stream_split" is False, 
+    the following encodings are supported.
```

Comment:
```suggestion
    Can only be used when when "use_dictionary" is set to False, and cannot be
    used in combination with "use_byte_stream_split".
```

(`use_byte_stream_split` is already False by default, in contrast to dictionary encoding)

Reply:
When `use_byte_stream_split` is enabled, the `column_encoding` is useless, should I note this in the doc of `use_byte_stream_split`?